src/quantum/azext_quantum/operations/workspace.py

```diff
@@ -43,6 +46,39 @@ def save(self, cmd):
             cmd.cli_ctx.config.set_value('quantum', 'group', self.resource_group)
             cmd.cli_ctx.config.set_value('quantum', 'workspace', self.name)
 
+def get_basic_quantum_workspace(location, info, storage_account):
+    qw = QuantumWorkspace()
+    # Use a default provider 
+    # Replace this with user specified providers as part of task:
+    # https://ms-quantum.visualstudio.com/Quantum%20Program/_workitems/edit/16184
+    prov = Provider()
+    prov.provider_id = "Microsoft"
+    prov.provider_sku = "Basic"
+    qw.providers = [prov]
+    # Allow the system to assign the workspace identity
+    qw.identity = QuantumWorkspaceIdentity()
+    qw.identity.type = "SystemAssigned"
+    qw.location=location
+    qw.storage_account = "/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Storage/storageAccounts/{2}".format(info.subscription, info.resource_group, storage_account)
```

Comment:
nit, for readability (ignore if this code has to support Python versions <3.6, when `f`-strings were introduced... 3.5 is out-of-support, but I'm not sure whether Azure CLI guarantees a particular Python version?)
```suggestion
    qw.storage_account = f"/subscriptions/{info.subscription}/resourceGroups/{info.resource_group}/providers/Microsoft.Storage/storageAccounts/{storage_account}"
```